packages/engine-core/src/data-proxy/DataProxyEngine.ts

```diff
@@ -56,7 +61,7 @@ export class DataProxyEngine extends Engine {
 
   private clientVersion: string
   readonly remoteClientVersion: Promise<string>
-  readonly headers: { Authorization: string }
+  readonly headers: { Authorization: string; 'prisma-capture-logs': boolean }
```

Comment:
```suggestion
  readonly headers: { Authorization: string; 'prisma-capture-logs': string }
```

headers are always strings, they can't be booleans (although maybe it would get implicitly converted it, but I'd still prefer to use correct types)